test/testcafe/spec/IdentifyWithThirdPartyIdps_spec.js

```diff
@@ -22,6 +24,17 @@ const mockOnlyOneIdp = RequestMock()
     res.setBody('<html><h1>A external IdP login page for testcafe testing</h1></html>');
   });
 
+const mockOnlyOneIdpAppUser = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(identifyOnlyOneIdpAppUser)
+  .onRequestTo('http://localhost:3000/sso/idps/facebook-idp-id-123?stateToken=inRUXNhsc6Evt7GAb8DPAA')
+  .respond(async (req, res) => {
+    // Delay response for 2 seconds so that we can verify the `success redirect` screen
+    await new Promise((r) => setTimeout(r, 2000));
```

Comment:
Avoid adding `delays` in our tests. Instead add a RequestLogger and check for network request was made or not..